Mathlib/SetTheory/Cardinal/Basic.lean

```diff
@@ -220,6 +220,11 @@ theorem lift_id' (a : Cardinal.{max u v}) : lift.{u} a = a :=
   inductionOn a fun _ => mk_congr Equiv.ulift
 #align cardinal.lift_id' Cardinal.lift_id'
 
+/-- A cardinal lifted to a lower or equal universe equals itself. -/
+@[simp, nolint simpNF]
+theorem lift_id'' (a : Cardinal.{max v u}) : lift.{u} a = a :=
+  inductionOn a fun _ => mk_congr Equiv.ulift
```

Comment:
Does this behave any differently from `lift_id'`? I think (in Lean 4) you need to explicitly specify the universes to rewrite with either of them.

Reply:
Good catch, I must have tricked myself into thinking I needed this. Removed.